src/niscope/metadata/functions.py

```diff
@@ -111,9 +111,9 @@
         'returns': 'ViStatus'
     },
     'AddWaveformProcessing': {
-        'codegen_method': 'private',
+        'codegen_method': 'public',
```

Comment:
I've been manually changing lines according to the diff of this file and the codegen results from the internal files because I noticed that functions.py has a lot of code (mostly related to hightime) that doesn't actually exist in the internal codegen file, and I'd rather not break everything. It would be great if you could clarify what I should do about this, since it looks like just copying the most up-to-date version of the codegen results wouldn't work.

Reply:
The best way to proceed is to post incremental PRs. Let's look at internal metadata and update first to a version prior to your changes but post `hightime`.

Reply:
See #1492.